x-pack/test/api_integration/apis/ml/modules/recognize_module.ts

```diff
@@ -44,7 +44,7 @@ export default ({ getService }: FtrProviderContext) => {
       user: USER.ML_POWERUSER,
       expected: {
         responseCode: 200,
-        moduleIds: ['apm_jsbase', 'apm_transaction', 'apm_nodejs'],
+        moduleIds: ['apm_jsbase', 'apm_nodejs'],
```

Comment:
Didn't we decide a long time ago to delete the language specific jobs because they didn't add value and weren't trained on enough language specific data?

Reply:
We will look into the best approach for a potential deprecation / legacy strategy for the APM RUM JavaScript and NodeJS jobs separately to the work on the transaction job. @sqren has confirmed that these two modules aren't referenced by the APM UI.